src/mpyl/schema/run_properties.schema.yml

```diff
@@ -17,7 +17,6 @@ properties:
     required:
       - run
       - versioning
-      - console
```

Comment:
I was actually thinking that with my pr I broke old tags again with the schema change. You will too with this since there is `additionalProperties: false`. Can you remove that for the `enabled` field as well and quickly get this change to the monorepo?

Reply:
There is no `additionalProperties` flag inside `build`, what am I missing?

Reply:
I thought you said that having it at the root applies it for the whole schema?

Reply:
I think we misunderstood each other, I'm pretty sure that's not the case. AFAIK each property needs to define if it allows additional sub-properties.